devimint/src/tests.rs

```diff
@@ -132,8 +133,13 @@ pub async fn latency_tests(dev_fed: DevFed, r#type: LatencyTest) -> Result<()> {
             // for enough time to catch up a session end
             let iterations = 30;
             let mut reissues = Vec::with_capacity(iterations);
+            let amount_per_iteration_msats =
+                // use a highest 2^-1 amount that fits, to try to use as many notes as possible
+                ((initial_balance_sats * 1000 / iterations as u64).next_power_of_two() >> 1) - 1;
```

Comment:
TIL about `next_power_of_two` :exploding_head: 

Note that this might blow up if we ever switch away from base-2 denominations as the default. The only commitment we made and assume everywhere is that denominations are b^x.

Reply:
Actually, it should fail gracefully, just not issuing as many notes anymore.